src/info/mod.rs

```diff
@@ -364,4 +373,135 @@ mod tests {
+                file_count: 100,
+            },
+            license: LicenseInfo {
+                license: String::from("info test license license"),
+            },
+            disabled_fields: vec![],
+            text_colors: TextColors {
+                title: DynColors::Ansi(AnsiColors::White),
+                tilde: DynColors::Ansi(AnsiColors::White),
+                underline: DynColors::Ansi(AnsiColors::White),
+                subtitle: DynColors::Ansi(AnsiColors::White),
+                colon: DynColors::Ansi(AnsiColors::White),
+                info: DynColors::Ansi(AnsiColors::White),
+            },
+            no_color_palette: true,
+            no_bold: true,
+            dominant_language: Language::Go,
+            ascii_colors: vec![DynColors::Ansi(AnsiColors::White)],
+        };
+        let expected_info =
+            [
+                "\u{1b}[37;1minfo test title git_username\u{1b}[0m \u{1b}[37;1m~\u{1b}[0m \u{1b}[37;1minfo test title git_version\u{1b}[0m\n",
+                "\u{1b}[37m----------------------------------------------------------\u{1b}[39m\n",
+                "\u{1b}[37mProject\u{1b}[0m\u{1b}[37m:\u{1b}[0m \u{1b}[37minfo test project repo_name (100 branches, 100 tags)\u{1b}[39m\n",
+                "\u{1b}[37mHEAD\u{1b}[0m\u{1b}[37m:\u{1b}[0m \u{1b}[37minfo test head head_refs short_commit_id (ref1, ref2)\u{1b}[39m\n",
+                "\u{1b}[37mPending\u{1b}[0m\u{1b}[37m:\u{1b}[0m \u{1b}[37minfo test pending pending_changes\u{1b}[39m\n",
+                "\u{1b}[37mVersion\u{1b}[0m\u{1b}[37m:\u{1b}[0m \u{1b}[37minfo test version version\u{1b}[39m\n",
+                "\u{1b}[37mCreated\u{1b}[0m\u{1b}[37m:\u{1b}[0m \u{1b}[37minfo test created creation_date\u{1b}[39m\n",
+                "\u{1b}[37mLanguage\u{1b}[0m\u{1b}[37m:\u{1b}[0m \u{1b}[37m\u{1b}[41m                          \u{1b}[49m\n",
+                "          \u{1b}[31m\u{25CF}\u{1b}[39m \u{1b}[37mGo (100.0 %)\u{1b}[39m \u{1b}[39m\n",
+                "\u{1b}[37mDependencies\u{1b}[0m\u{1b}[37m:\u{1b}[0m \u{1b}[37minfo test dependencies dependencies\u{1b}[39m\n",
+                "\u{1b}[37mAuthor\u{1b}[0m\u{1b}[37m:\u{1b}[0m \u{1b}[37m\u{1b}[37m100% author1 name 100\u{1b}[39m\u{1b}[39m\n",
+                "\u{1b}[37mLast change\u{1b}[0m\u{1b}[37m:\u{1b}[0m \u{1b}[37minfo test last_change last_change\u{1b}[39m\n",
+                "\u{1b}[37mRepo\u{1b}[0m\u{1b}[37m:\u{1b}[0m \u{1b}[37minfo test repo repo_url\u{1b}[39m\n",
+                "\u{1b}[37mCommits\u{1b}[0m\u{1b}[37m:\u{1b}[0m \u{1b}[37minfo test commits number_of_commits\u{1b}[39m\n",
+                "\u{1b}[37mLines of code\u{1b}[0m\u{1b}[37m:\u{1b}[0m \u{1b}[37m100\u{1b}[39m\n",
+                "\u{1b}[37mSize\u{1b}[0m\u{1b}[37m:\u{1b}[0m \u{1b}[37minfo test size repo_size (100 files)\u{1b}[39m\n",
+                "\u{1b}[37mLicense\u{1b}[0m\u{1b}[37m:\u{1b}[0m \u{1b}[37minfo test license license\u{1b}[39m\n" ].join("");
```

Comment:
What matters here (IMO) are the information displayed, not the formatting.

In that regard, I think asserting with a `contains` instead of looking for an exact match would suffice and improve the readability quite a lot. cf. cc7a59c for an example.

Reply:
@o2sh , thank you, will reduce to important aspects using contains and example